raft/raft.go

```diff
@@ -138,7 +138,8 @@ type Raft struct {
 	// number of ticks since it reached last heartbeatTimeout.
 	// only leader keeps heartbeatElapsed.
 	heartbeatElapsed int
-	// number of ticks since it reached last electionTimeout
+	// number of ticks since it received last heartbeat.
+	// followers and candidates keeps electionElapsed.
```

Comment:
Besides `hearbeat`, other valid messages from the leader (like `AppendEntires`, `Snapshot`, etc) can also reset `electionTimeout`, maybe we can just change the comment to the sane as `raft-rs`:

https://github.com/tikv/raft-rs/blob/c7c230f63c59d408176e9288966da00c579ef4ba/src/raft.rs#L210-L213

Reply:
Cool, it's surely better, `valid message` best describes it.